apps-rendering/src/components/Paragraph/index.tsx

```diff
@@ -81,8 +81,9 @@ const styles = (
 	return css`
 		${body.medium()}
 		overflow-wrap: break-word;
-		margin: 0 0 ${remSpace[3]};
+		margin: 0 0 0.875rem;
 		color: ${text.paragraph(format)};
+		line-height: 1.4;
```

Comment:
@guardian/client-side-infra if this is the new standard from Design should the `body` typography API in Source be updated to reflect this?

Reply:
Thanks for flagging, Jamie. Ideally, yes.

Reply:
There is a ticket to implement this in the DS Advocates board: https://github.com/orgs/guardian/projects/104?pane=issue&itemId=30696868

Reply:
👍 nice, thanks. I've added #8386 to follow-up once that's released.

Reply:
@DanielCliftonGuardian I think we probably want to avoid a situation where we're adding line-height directly to platforms - its just means that future changes to Source Foundations then won't be reflected in the code. 

If you're around tomorrow (Thursday) let's have a chat about how best we move this one forwards.

Reply:
Cool thanks. I'm chatting to Ravi about implications for Commercial but hope to have a Source release with this in ASAP.

Reply:
These values are now available in source-foundations v13.0.0

Reply:
I've merged #8426 into this branch; this change has been removed and is now provided by the call to `body.medium()` a few lines above.